dynamo.code-workspace

```diff
@@ -8,7 +8,8 @@
         "rust-analyzer.linkedProjects": [
             "Cargo.toml",
             "launch/dynamo-run/Cargo.toml",
-            "lib/bindings/python/Cargo.toml"
+            "lib/bindings/python/Cargo.toml",
+            "lib/runtime/examples/Cargo.toml"
```

Comment:
these examples should be run by ci.  they currently are not.  

we might need to update the top-level Cargo.toml and/or CI configs